csharp_dotnetcore/12.NLP-With-LUIS/Luis-Bot.ruleset

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Consider just having this be LuisBot.ruleset as the file name. HAving the "-" everywhere is a problem as namespaces and class names don't support that character.

Reply:
Good idea, Renamed project